### PR TITLE
dates/mlh banners

### PR DESCRIPTION
**Pull request by MWedee**
Changed some dates to Feb 11-12 / 2023 from previous Oct/Jan 22 dates. 
Removed MLH banner from all pages as they were outdated/we are  not MLH this season.

### CODE REVIEW

File: 200.html [hunk 1]
```diff
@@ -1,9 +1,148 @@
...
+    <link
+      data-n-head="true"
+      rel="icon"
+      type="image/x-icon"
+      href="/owlhacks-icon.png"
+    />
+    <link rel="preload" href="/_nuxt/a9319e3cd0013af34655.js" as="script" />
+    <link rel="preload" href="/_nuxt/d368e7381e4970061e75.js" as="script" />
+    <link rel="preload" href="/_nuxt/93ab85c19c5f7acf8687.js" as="script" />
+    <link rel="preload" href="/_nuxt/54c6922fb61b4818893d.js" as="script" />
```
Comment (zachgoncalves) [inline 200.html:28]: i don't think this site uses nuxt stuff anymore. i guess it's carryover from when it was converted from before. probably worth getting rid of it.

**Review by zachgoncalves — COMMENTED** [review]
are you taking over OwlHacks? i can probably add you as an admin to the repo, though I am not an organization admin.

File: 200.html (hunk 1 above)
Comment (MWedee) [reply 200.html:28]: > are you taking over OwlHacks? i can probably add you as an admin to the repo, though I am not an organization admin.

yeah I'm taking it over or 2023! Have to make some changes so the website is up to date to share with sponsors asap! 👍 

> i don't think this site uses nuxt stuff anymore. i guess it's carryover from when it was converted from before. probably worth getting rid of it.

I'm just getting familiar with the website now and I haven't touched much besides some text in the html files and commenting out/in some stuff. We aren't MLH certified this year so had to take out those banners. Besides that I just used a formatter from vscode on a few files.

**Review by lawja — APPROVED** [review]
nice. godspeed.

**Comment by lawja** [comment]
it's been ages since I've used github, so idk if I merge or if the PR author merges, but lgtm. Waiting on the github pages signal to pass, but seems fine. lmk if you need me to merge / cc me on any other PRs that need review, I'll try and approve to unblock everything. In the meantime I'll try and give you admin rights to the repo so you don't need dinosaurs like Zach and I to allow you to get work done

File: 200.html (hunk 1 above)
Comment (zachgoncalves) [reply 200.html:28]: Sounds good! You can do whatever you want to the site - redesign it, go with whats there, whatever - have fun!

**Comment by zachgoncalves** [comment]
> it's been ages since I've used github, so idk if I merge or if the PR author merges, but lgtm. Waiting on the github pages signal to pass, but seems fine. lmk if you need me to merge / cc me on any other PRs that need review, I'll try and approve to unblock everything. In the meantime I'll try and give you admin rights to the repo so you don't need dinosaurs like Zach and I to allow you to get work done

Stealing my thunder...

**Review by zachgoncalves — APPROVED** [review]

**Comment by MWedee** [comment]
Hey let me know if you guys are able to do the final checks and do the merge. Thanks

**Pull request merged**